src/main/kotlin/com/criptext/mail/utils/generaldatasource/workers/LogoutWorker.kt

```diff
@@ -85,7 +81,11 @@ class LogoutWorker(
                         storage.clearAll()
                     }
                     if(!shouldDeleteAllData) {
-                        storage.putString(KeyValueStorage.StringKey.LastLoggedUser, activeAccount.recipientId)
+                        val storageLastUser = storage.getString(KeyValueStorage.StringKey.LastLoggedUser, "")
+                        val loggedOutAccounts = if(storageLastUser.isEmpty()) mutableListOf() else
```

Comment:
This portion of code is repeated too many times. Create a function inside `storage` maybe.